samples/bluetooth/peripheral_hids_keyboard/child_image/hci_rpmsg.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+CONFIG_SDC_SLAVE_COUNT=2
```

Comment:
Instead of setting `CONFIG_SDC_SLAVE_COUNT=2` I would consider setting:
`CONFIG_BT_CENTRAL=n`. That will give greater size reductions. It will also ensure that two slave links are supported

Reply:
Ok. I also added `CONFIG_BT_CONN_MAX=2` I think it is more descriptive for user now